nbs/01_augment.ipynb

```diff
@@ -49,6 +49,9 @@
    "outputs": [],
```

Comment:
<p>The verify_pipeline function defined here should be instead on the Pipelines section, that is a couple of cells below. Also, it would be very helpful to include some examples, and you can add tests that it works using <code>test_warns</code> function  from fastcore.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rbracco/fastai2_audio/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='194'/>